tests/MSBuildDeviceIntegration/Tests/AotProfileTests.cs

```diff
@@ -22,7 +22,10 @@ public void BuildBasicApplicationAndAotProfileIt ()
 		{
 			AssertHasDevices ();
 
-			var proj = new XamarinAndroidApplicationProject () { IsRelease = true };
+			var proj = new XamarinAndroidApplicationProject () {
+				IsRelease = true,
+				PackageName = GeneratePackageName (),
```

Comment:
When do we want to set this? For all "ondevice" tests?

Reply:
I would say yes for all on Device tests, unless you manually specify a package name yourself. There are some on device tests which use a specific packageName.